Ping/handler.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
+import * as express from "express";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "@pagopa/io-functions-commons/dist/src/utils/request_middleware";
+import {
+  IResponseErrorInternal,
+  IResponseSuccessJson,
+  ResponseSuccessJson
+} from "@pagopa/ts-commons/lib/responses";
+
+import { verifyJWTMiddleware } from "../utils/auth-jwt";
+import { JWTConfig } from "../utils/config";
+
+import { ServiceStatus } from "../generated/definitions/external/ServiceStatus";
+
+type InfoHandler = () => Promise<
+  IResponseSuccessJson<ServiceStatus> | IResponseErrorInternal
+>;
+
+export const PingHandler = (): InfoHandler => (): Promise<
+  IResponseSuccessJson<ServiceStatus> | IResponseErrorInternal
+> =>
+  new Promise(resolve => {
+    resolve(
+      ResponseSuccessJson<ServiceStatus>({
+        message: "Function IO Web Profile is up and running"
+      })
+    );
+  });
```

Comment:
Just a tip:
( PS: We need to add  `import * as T from "fp-ts/Task";` )

```suggestion
  T.of(
    ResponseSuccessJson<ServiceStatus>({
      message: "Function IO Web Profile is up and running"
    })
  )();
```